src/test/java/io/radanalytics/streaming/amqp/JavaAMQPServerStreamSuite.java

```diff
@@ -15,17 +15,20 @@
  * limitations under the License.
  */
 
-package org.apache.spark.streaming.amqp;
+package io.radanalytics.streaming.amqp;
 
+import org.apache.qpid.proton.message.Message;
 import org.apache.spark.SparkConf;
 import org.apache.spark.api.java.function.Function;
 import org.apache.spark.storage.StorageLevel;
+import org.apache.spark.streaming.amqp.AMQPUtils;
```

Comment:
Why AMQPUtils is still in the "old" package ?

Reply:
I wanted to put everything into `io.radanalytics.streaming.amqp` package, but the original code was using the package protected visibility. It was in the same package as spark's stuff, so it was able to access those classes such as (specifically for `AMQPUtils`):

Reply:
If it was just exposing the fields of a class that is marked as package protected, I could have used something like this:.
https://github.com/radanalyticsio/silex/blob/develop/src/main/scala/io/radanalytics/silex/som/hacks.scala

But the code in this repo extends those package protected classes, or use them on a type declaration places and I didn't fined a way to do that. Using reflection and changing the visibility would be bad, imho.